hackerRank/a-very-big-sum/a-very-big-sum.js

```diff
@@ -0,0 +1,11 @@
+const aVeryBigSum = (ar) => {
+  let addBigSum = 0;
+
+  ar.forEach((element) => {
+    addBigSum += element;
+  });
+
```

Comment:
Use this opportunity to learn about array.reduce, it would be nice to use that instead.

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

Reply:
Thank for the review @Yinkusadel 
Please take a look at @teckcipher Requested changes up here, I just started using the forEach here, Initially I wasn't using forEach but I refactor my code to forEach